### PR TITLE
hack/cherry_pick_pull.sh: Allow multiple pulls

### PR DESCRIPTION
**Pull request by zmerlynn**
This solves some common A-then-immediate-A' cases that appear
frequently on head. There's a workaround, but it's a hack.

### CODE REVIEW

**Comment by zmerlynn** [comment]
cc @thockin, since you asked for it.

File: hack/cherry_pick_pull.sh [hunk 1]
```diff
@@ -28,12 +28,13 @@ cd "${KUBE_ROOT}"
 declare -r STARTINGBRANCH=$(git symbolic-ref --short HEAD)
 declare -r REBASEMAGIC="${KUBE_ROOT}/.git/rebase-apply"
 
-if [[ "$#" -ne 2 ]]; then
-  echo "${0} <pr-number> <remote branch>: cherry pick <pr> onto <remote branch> and leave instructions for proposing pull request"
+if [[ "$#" -lt 2 ]]; then
+  echo "${0} <pr-number>... <remote branch>: cherry pick one or more <pr> onto <remote branch> and leave instructions for proposing pull request"
```
Comment (thockin) [inline hack/cherry_pick_pull.sh]: totally optional: would it be easier to put the branch first, so varargs are last?

**Comment by thockin** [comment]
LGTM

I am oncall, so tell me about the commandline if you want to change it.  Else I will assume to commit this Monday AM.

File: hack/cherry_pick_pull.sh (hunk 1 above)
Comment (zmerlynn) [inline hack/cherry_pick_pull.sh]: I guess it's new enough, internal enough, etc, that a "breaking" change in ordering is fine. :) (We also detect non-branchlike things pretty easily, so a misordering isn't fatal.)

I'll flip it. The array nonsense is painful, and shift is easy. AFK right now, will push later.

Comment (thockin) [inline hack/cherry_pick_pull.sh]: ping me when done

**Comment by zmerlynn** [comment]
@thockin: PTAL

**Comment by thockin** [comment]
LGTM

**Comment by eikaas** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  tapiohush@gmail.com
    Domain eikaas.no has exceeded the max emails per hour (34/30 (113%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext8.iad.github.net ([192.30.252.199]:49201 helo=github-smtp2a-ext-cp1-prd.iad.github.net)
	by cpanel30.proisp.no with esmtps (TLSv1.2:DHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.85)
	(envelope-from <noreply@github.com>)
	id 1ZEgpI-004Cre-LH
	for robin@eikaas.no; Mon, 13 Jul 2015 18:43:34 +0200
Date: Mon, 13 Jul 2015 09:43:31 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1436805811;
	bh=3a7XJze9iOORHewnHBIFsMf735rKgYCgYgXIGRzAB04=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=cUqNK+cBaXKii9ibZJDsJ/WVM0qHf+6crVl+6ffqhnY7YM9kuR2jNhakXbg8/zKq8
	 SHGsYgrZi9goN3s06Mj1x+nVIy1/NerjsKAfgMIFspb4kl4xk7T0svKgkW+Q9HZse+
	 Lbiq2I1xWgVJLPq1Qv4kKupYH60I8bVGdoCQSpc4=

**Pull request merged**